go/glide.yaml

```diff
@@ -23,11 +23,13 @@ import:
 - package: github.com/go-ini/ini
   version: ^1.21.0
 - package: github.com/projectcalico/libcalico-go
-  version: smc-merged-fixes
-  repo: https://github.com/fasaxc/libcalico-go.git
+  version: k8s-backend 
+  repo: https://github.com/caseydavenport/libcalico-go.git
   subpackages:
   - lib
 - package: github.com/Sirupsen/logrus
   version: ^0.10.0
 - package: github.com/kardianos/osext
 - package: github.com/mipearson/rfw
+- package: github.com/gogo/protobuf/proto
```

Comment:
I'll try removing this - I needed it when I first started building, but likely because it was pulling in from somewhere else on my system.